packages/store/ts/library/config/index.ts

```diff
@@ -96,15 +107,31 @@ export type FullTableConfig<
   schema: FullSchemaConfig<UserTypes>;
 };
 
+export interface ExpandTableConfig<T extends TableConfig<string, string>, TableName extends string>
+  extends OrDefaults<
+    T,
+    {
+      directory: "tables";
+      name: TableName;
+      tableIdArgument: false;
+      storeArgument: false;
+      // TODO should dataStruct conditional default be expanded?
+      dataStruct: boolean;
```

Comment:
probably shouldn't be expanded, since its value is conditional on the number of schema fields, which we can't replicate in typescript type logic afaik

Reply:
agree, change the comment to reflect this